packages/eslint-plugin-clarity-migration/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@clr/eslint-plugin-clarity-migration",
```

Comment:
Do we need to publish this as `eslint-plugin-clarity-migration` instead of with `@clr` prefix?

Reply:
Right now the package is configured to be published under the `@clr` scope - `@clr/eslint-plugin-clarity-migration`. If we don't want that, we can also publish it as `eslint-plugin-clarity-migration`.